src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedString = sourceString
+    .split(';')
+    .map((str) => str.split(':'))
+    .map((str) => str.map((value) => value.trim()))
+    .map((str) => str.filter((value) => value !== ''))
+    .filter((str) => str.length !== 0)
+    .reduce((object, item) => {
+      object[item[0]] = item[1];
+
+      return object;
+    }, {});
+
+  return splittedString;
```

Comment:
The function `convertToObject` is using a combination of `map`, `filter`, and `reduce` to process the input string, which is good practice according to the checklist. However, the variable name `splittedString` is not descriptive of the result it holds, which is an object, not a string. Consider renaming it to something more descriptive, like `styleObject` to improve code readability.

Reply:
The same as in previous comment